lib/auth/authclient/clt.go

```diff
@@ -809,6 +809,11 @@ func (c *Client) UpsertUserNotification(ctx context.Context, notification *notif
 	return nil, trace.NotImplemented(notImplementedMessage)
 }
 
+// ListUserNotificationStatesForAllUsers not implemented: can only be called locally.
```

Comment:
We have quite a few other methods only intended for internal use which also being exposed here as not implemented, I think refactoring the whole notifications service interface to achieve this would be beyond the scope of this PR